app/gui2/ydoc-server/edits.ts

```diff
@@ -12,6 +12,29 @@ import { assert } from '../shared/util/assert'
 import { IdMap, ModuleDoc, type VisualizationMetadata } from '../shared/yjsModel'
 import * as fileFormat from './fileFormat'
 
+/**
+ * The simulated metadata of this size takes c.a. 1 second on my machine. It should be quite
+ * bearable, even on slower machines.
```

Comment:
1 second is still quite excessive, but we can have that temporarily. Eventually we need a better solution though (either by structural diffing or by somehow avoiding diffing altogether).